### ISSUE TITLE
Documentation error for day attribute of categories?

### ISSUE THREAD

**Issue by cbrxyz** [amazingmarvin/MarvinAPI#11]
The documentation for the `day` attribute of categories says the attribute will either be a date or null/undefined. It should not be "unassigned." However, when viewing categories in my db, there is an object with `day = "unassigned"`. Other categories do have `day = None`, like the documentation states, but one object has the `unassigned` value.

**Comment by amazingmarvin** [comment]
Oops, sorry to hear this. Originally projects couldn't be scheduled, and tasks without a schedule date had `day=null`. But this broke the `pouchdb-find` library so I had to take `day="unassigned"`. We quickly got rid of `pouchdb-find` since I had other issues with it, but this unfortunate data stuck around. When we added the ability to schedule projects, I """corrected this design error""" by making it so that unscheduled projects had `day=null`. I guess there's some use case that is causing projects to get `day="unassigned"`... maybe dragging them into a smart list that checks for unscheduled items?

So it's unfortunate but Marvin checks for unscheduled tasks and projects by checking if !day or day==="unassigned". Sorry about this! I'll make a note in the documentation linking to this explanation.
